### ISSUE TITLE
more seperation layer

### ISSUE THREAD

**Issue by infofromca** [Yu-Core/SwashbucklerDiary#41]
**Is your feature request related to a problem? Please describe.**
too couple

**Describe the solution you'd like**

in SwashbucklerDiary.Maui project
**Describe alternatives you've considered**
make SwashbucklerDiary.Maui  as nuget

**Additional context**
the same priciple in also other projects maybe

**Comment by Yu-Core** [comment]
Sorry, my English is not good
Do you mean to move the content of SwashbucklerDiary.Repository to SwashbucklerDiary.Maui
SwashbucklerDiary.WebAssembly also needs to reference SwashbucklerDiary.Repository
SwashbucklerDiary.WebAssembly is a Blazor WebAssembly project

**Comment by infofromca** [comment]
1--分层， 比如：
SwashbucklerDiary.Maui.Core
SwashbucklerDiary.WebAssembly.Core
SwashbucklerDiary.Rcl
2-- 成为： nuget
3--目的： 其它projects 使用。 因为有很多很好的codes.

**Comment by infofromca** [comment]
> Do you mean to move the content of SwashbucklerDiary.Repository to SwashbucklerDiary.Maui
no, 去除SwashbucklerDiary.Repository， 新加SwashbucklerDiary.Maui.Core。 SwashbucklerDiary.Maui.Core 可在其它项目使用

**Comment by Yu-Core** [comment]
明白你的意思了，把项目一些公用的东西抽离出来做成nuget包（或者是模板？），抽离出来的话有点困难，需要一些时间

**Comment by infofromca** [comment]
我可帮忙， 新的 Project 的版 可 否 为 MIT, OR BSD-3
